### PR TITLE
No need to run `FreshRunReq`, this is normal operation for the PC.

### PR DESCRIPTION
**Pull request by dragos**

### CODE REVIEW

**Comment by dragos** [comment]
retest this please

File: org.scala-ide.sdt.core/src/org/scalaide/core/compiler/IScalaPresentationCompiler.scala [hunk 1]
```diff
@@ -367,6 +367,10 @@ object IScalaPresentationCompiler extends HasLogger {
                 Thread.currentThread().interrupt()
                 None
 
+              case Right(f: FreshRunReq)          =>
+                logger.info("FreshRunReq in ask:\n" + f)
```
Comment (sschaef) [inline org.scala-ide.sdt.core/src/org/scalaide/core/compiler/IScalaPresentationCompiler.scala]: `info` has an overload, which takes in its second argument a throwable. Please choose that one.

Comment (dragos) [inline org.scala-ide.sdt.core/src/org/scalaide/core/compiler/IScalaPresentationCompiler.scala]: Note that `FreshRunRequest` has no stack trace, but I can use the overload nonetheless

**Comment by sschaef** [comment]
Generally, LGTM. How did you find out about this issue? Was there a bug somewhere?

**Comment by dragos** [comment]
I started to see entries in the Eclipse log (there's a catch-all at the end, who'd route the exception to the Eclipse error log).

**Pull request merged**